File: engine/engine-model/src/main/scala/tools/packet/cashshop/operation/RefundCashItem.scala
package tools.packet.cashshop.operation

import client.inventory.Item
import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RefundCashItem(private var _item: Item, private var _maplePoints: Int) extends PacketInput {
  def item: Item = _item

  def maplePoints: Int = _maplePoints

  override def opcode(): SendOpcode = SendOpcode.CASHSHOP_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/RefundCashItem.java
package tools.packet.cashshop.operation;

import client.inventory.Item;
import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RefundCashItem(Item item, Integer maplePoints) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}